### PR TITLE
Generate button naming issue

### PR DESCRIPTION
**Pull request by gtico80**
For Issue #9 a solution based on the fact that the button change from generate to validate if a file has been uploaded.

### CODE REVIEW

**Comment by libremente** [comment]
Thank you @gtico80 for the PR! We will evaluate it ASAP and let you know.

**Comment by libremente** [comment]
Ok, @gtico80 I tested your solution. I believe it works as expected and as mentioned in #9. However, I think it would be better to redefine the whole flow (as explained in #5) as such:
1. upload a `publiccode.yml`.
2. **_Call the validator_** -> Immediately display the possible errors.
3. Populate the forms with the info from the file.

At this point, the user has a clear overview of the situation (the errors are shown, if any) and it's possible to fill the missing info or correct the problems **through the forms**. As such, in this use case I believe that the `Validate` button text may be misleading and it should be reverted to `Generate`.

So, to sum up, I guess that by inserting the validator call as soon as the upload is correctly completed we can solve #5 and make #9 useless. What do you think about this? @sebbalex

**Comment by sebbalex** [comment]
I agree with @libremente point of view, button text in that moment could be misleading and confusing.
As well solving #5 will solve both issues. I would go in this way.

**Comment by libremente** [comment]
Hi @gtico80 and thank you for your contribution! We decided to merge your code in our `release-1.1` branch since it solved the issue as you mentioned. 
Good job! :1st_place_medal:
